### PR TITLE
k8s: support setting cluster and service subnets on create.

### PR DESCRIPTION
**Pull request by andrewsomething**
https://docs.digitalocean.com/products/kubernetes/how-to/create-clusters/#create-with-vpc-native

CC: @gottwald

### CODE REVIEW

**Review by loosla — APPROVED** [review]
Looks good to me :rocket:

File: digitalocean/kubernetes/resource_kubernetes_cluster_test.go [hunk 1]
```diff
@@ -759,6 +759,29 @@ func TestAccDigitalOceanKubernetesCluster_DestroyAssociated(t *testing.T) {
...
+		CheckDestroy:      testAccCheckDigitalOceanKubernetesClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDigitalOceanKubernetesConfigVPCNative(testClusterVersionLatest, rName),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckDigitalOceanKubernetesClusterExists("digitalocean_kubernetes_cluster.foobar", &k8s),
+					resource.TestCheckResourceAttr("digitalocean_kubernetes_cluster.foobar", "name", rName),
+					resource.TestCheckResourceAttr("digitalocean_kubernetes_cluster.foobar", "region", "nyc1"),
+					resource.TestCheckResourceAttr("digitalocean_kubernetes_cluster.foobar", "cluster_subnet", "192.168.0.0/20"),
+					resource.TestCheckResourceAttr("digitalocean_kubernetes_cluster.foobar", "service_subnet", "192.168.16.0/24"),
```
Comment (gottwald) [inline digitalocean/kubernetes/resource_kubernetes_cluster_test.go:778]: Can you make the service net a /22? We want to put some limitations in the API to enforce certain minimum sizes.

**Review by gottwald — COMMENTED** [review]
Mostly LGTM already, but would like to make that one small change.

File: digitalocean/kubernetes/resource_kubernetes_cluster_test.go (hunk 1 above)
Comment (andrewsomething) [reply digitalocean/kubernetes/resource_kubernetes_cluster_test.go:778]: Updated in https://github.com/digitalocean/terraform-provider-digitalocean/pull/1253/commits/c6eced9b8a36303fcfedeeca782e5505b79b6ef4

**Review by gottwald — APPROVED** [review]

**Pull request merged**